src/components/dashboard/Dashboard.js

```diff
@@ -497,6 +505,49 @@ const Dashboard = props => {
 
   const avatarSource = avatar || unknownProfile
 
+  const memoFeedList = useMemo(() => {
```

Comment:
you don't need this. all you need is just to memoize debounced onScroll  handler
```

const handleScroll = useMemo(() => debouce(({ nativeEvent }) => {
 //
}), [feeds, headerLarge]);
// ...
onScroll={handleScroll}
```